platform/datacollector/src/main.rs

```diff
@@ -14,7 +19,7 @@ async fn health_handler() -> impl Responder {
 #[get("/data")]
 async fn data_handler() -> impl Responder {
```

Comment:
I would make this 

```rust
async fn data_handler(_event: Event) -> Event {
```

and then return `response_event` as is, it still implements the Responder to satisfy actix, but then is also a cloud event plus http response